src/mscorlib/ref/mscorlib.cs

```diff
@@ -7344,7 +7344,7 @@ public enum AssemblyNameFlags
     [System.Runtime.InteropServices.ComVisibleAttribute(true)]
     public class AssemblyNameProxy : System.MarshalByRefObject
     {
-        internal AssemblyNameProxy() { }
+        public AssemblyNameProxy() { }
```

Comment:
Don't you also need the `GetAssemblyName` name method?

Also, even though this type is in System.Reflection namespace, it is really part of remoting - it is a trivial remoting forwarder to AsssemblyName. It should we better implemented in corefx where all other dummy stubs for remoting types will be, I assume.